app/controllers/features_controller.rb

```diff
@@ -4,7 +4,7 @@ class FeaturesController < ApplicationController
   def index
     @title = "Features"
     @features = Node.where(type: 'feature')
-      .paginate(page: params[:page])
+    @features = Node.sort_features(@features, params[:sort]).paginate(page: params[:page])
```

Comment:
Hi, @Manasa2850 , maybe we should rename one of these variables, what do you think?

Everything else looks great thanks  a bunch for working on this

Reply:
Hi @cesswairimu, sure I'll change that. Thank you!